dashboard/config/routes.rb

```diff
@@ -620,6 +620,7 @@
       get 'projects/gallery/public/:project_type/:limit(/:published_before)', to: 'projects/public_gallery#index', defaults: {format: 'json'}
 
       get 'projects/personal', to: 'projects/personal_projects#index', defaults: {format: 'json'}
+      get 'libraries/classmates/all', to: 'section_libraries#get_classmate_published_libraries', defaults: {format: 'json'}
 
```

Comment:
This looks like a good spot for this route!

Reply:
🎉